recipes/discid/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "discid" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/discid-{{ version }}.tar.gz
+  sha256: cd9630bc53f5566df92819641040226e290b2047573f2c74a6e92b8eed9e86b9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - libdiscid
```

This should make it noarch but installable only on Linux and macOS.

```suggestion
    - libdiscid
    - __unix  # [unix]
```